easybuild/easyconfigs/w/Wannier90/wannier90-3.1.0-gfort.patch

```diff
@@ -0,0 +1,24 @@
+--- config/make.inc.gfort.orig	2020-03-05 13:41:10.000000000 -0500
++++ config/make.inc.gfort	2022-06-02 09:22:27.141245863 -0400
+@@ -4,11 +4,11 @@
+ F90 = gfortran
+ 
+ #Add next two lines for parallel postw90. Choose the correct name of the mpi f90 wrapper
+-#COMMS  = mpi
+-#MPIF90 = mpgfortran #mpif90
++COMMS  = mpi
++MPIF90 = mpif90
+ 
+-FCOPTS = -O3
+-LDOPTS =
++FCOPTS = -O2
++LDOPTS = -O2
+ 
+ #Next two lines are good for debugging
+ #FCOPTS = -fstrict-aliasing  -fno-omit-frame-pointer -fno-realloc-lhs -fcheck=bounds,do,recursion,pointer -ffree-form -Wall -Waliasing -Wsurprising -Wline-truncation -Wno-tabs -Wno-uninitialized -Wno-unused-dummy-argument -Wno-unused -Wno-character-truncation -O1 -g -fbacktrace
+@@ -34,4 +34,4 @@
+ #   sudo apt-get install libblas-dev liblapack-dev
+ # on Ubuntu
+ #=======================
+-LIBS = -llapack -lblas
++LIBS = -lflexiblas
```

Comment:
Use `$(LIBLAPACK) $(LIBBLAS)` to be more generic

Reply:
done